src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.Setup.csproj

```diff
@@ -56,7 +56,8 @@
           AfterTargets="Build"
           BeforeTargets="SwixBuild"
           DependsOnTargets="_SetSwrFilePath;InitializeDesktopCompilerArtifacts;_PrepareDesktopCompilerArtifactsForOptimization;ApplyOptimizations"
-          Outputs="$(_SwrFilePath)">
+          Outputs="$(_SwrFilePath)"
+          Condition="'$(DotNetBuildFromSource)' != 'true'">
```

Comment:
I'm surprised this is running in source build at all, it feels like swr packaging should have been skipped at a much earlier point?

Reply:
It's running because of `AfterTargets="Build"`.